build_docker.sh

```diff
@@ -1 +0,0 @@
-docker build -t mottosso/qt.py27 -f Dockerfile-py2.7 .
```

Comment:
The build step, potentially, now that it's taking so much longer and isn't relevant for local testing anymore (as they'll be pulling instead of building).

For testing and running however, yeah, I run these all the time and have very little memory for the exact names of images and command-line flags. Also as I said it simplifies the docs, see "Running Tests": https://github.com/mottosso/Qt.py#developer-guide